src/site/tests/support/axe.js

```diff
@@ -0,0 +1,62 @@
+import { run } from 'axe-core';
+
+const logViolations = violations => {
+  /* eslint-disable no-console */
+  console.log(
+    `\n${violations.length} Accessibility Violation${
+      violations.length === 1 ? ' Was' : 's Were'
+    } Detected`,
+  );
+
+  violations.forEach((violation, violationIdx) => {
+    console.log(`\nAxe Violation ${violationIdx + 1}:\n`, violation);
+
+    violation.nodes.forEach((node, nodeIdx) => {
+      console.log(`\nNode ${nodeIdx + 1}:\n`, node);
+    });
+  });
+
+  /* eslint-enable no-console */
+};
+
+const axeCheck = container => {
+  const options = {
+    runOnly: {
+      type: 'tag',
+      values: ['section508', 'wcag2a', 'wcag2aa'],
```

Comment:
Do we need 2a and 2aa both listed? wouldn't aa include a?

Reply:
@1Copenut , I think this is a question that you're better suited to answer than I am. What do you think?